concrete/src/Updater/Migrations/Migrations/Version20180227035239.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180227035239 extends AbstractMigration
```

Comment:
@aembler This is a repeatable migration, isn't it?

Reply:
Yes. I have added the interface in a new commit.